watertap/costing/zero_order_costing.py

```diff
@@ -141,97 +146,71 @@ def build_process_costs(self):
             doc="Insurance and taxes costs - based on aggregate capital costs",
         )
-        self.total_fixed_operating_cost = pyo.Var(
-            initialize=0,
-            units=self.base_currency / self.base_period,
-            doc="Total fixed operating costs",
+        self.factor_maintenance_labor_chemical = pyo.Expression(
+            expr=self.salaries_percent_FCI
+            + self.benefit_percent_of_salary * self.salaries_percent_FCI
+            + self.maintenance_costs_percent_FCI
+            + self.laboratory_fees_percent_FCI
+            + self.insurance_and_taxes_percent_FCI,
+            doc="Maintenance-labor-chemical factor [fraction of equipment cost/year]",
         )
 
-        self.salary_cost_constraint = pyo.Constraint(
-            expr=self.salary_cost
-            == self.aggregate_capital_cost * self.salaries_percent_FCI
-        )
-        self.benefits_cost_constraint = pyo.Constraint(
-            expr=self.benefits_cost == self.salary_cost * self.benefit_percent_of_salary
-        )
-        self.maintenance_cost_constraint = pyo.Constraint(
-            expr=self.maintenance_cost
-            == self.aggregate_capital_cost * self.maintenance_costs_percent_FCI
-        )
-        self.laboratory_cost_constraint = pyo.Constraint(
-            expr=self.laboratory_cost
-            == self.aggregate_capital_cost * self.laboratory_fees_percent_FCI
-        )
-        self.insurance_and_taxes_cost_constraint = pyo.Constraint(
-            expr=self.insurance_and_taxes_cost
-            == self.aggregate_capital_cost * self.insurance_and_taxes_percent_FCI
+        self.maintenance_labor_chemical_operating_cost = pyo.Expression(
+            expr=self.factor_maintenance_labor_chemical * self.aggregate_capital_cost,
+            doc="Maintenance-labor-chemical operating cost",
         )
 
-        self.total_fixed_operating_cost_constraint = pyo.Constraint(
-            expr=self.total_fixed_operating_cost
-            == self.aggregate_fixed_operating_cost
-            + self.salary_cost
-            + self.benefits_cost
-            + self.maintenance_cost
-            + self.laboratory_cost
-            + self.insurance_and_taxes_cost
+        self.total_fixed_operating_cost = pyo.Expression(
+            expr=self.aggregate_fixed_operating_cost
+            + self.maintenance_labor_chemical_operating_cost,
+            doc="Total fixed operating costs",
         )
 
         # Other variable costs
         self.total_variable_operating_cost = pyo.Expression(
-            expr=self.aggregate_variable_operating_cost
-            + sum(self.aggregate_flow_costs[f] for f in self.used_flows)
-            * self.utilization_factor,
+            expr=(
+                self.aggregate_variable_operating_cost
+                + sum(self.aggregate_flow_costs[f] for f in self.used_flows)
+                * self.utilization_factor
+            )
+            if self.used_flows
```

Comment:
Curious why this if/else statement is needed here.

Reply:
if `self.used_flows` is empty, the the `sum` on line 209 will be 0, but its units will be incorrect.

There is similar logic in the `WaterTAPCosting` package, but it wasn't here. I am guessing it hasn't come up in a ZO flowsheet, but it conceivably could. So this was added primarily to have similar code in both packages which is easier to integrate.